finetune/lora.py

```diff
@@ -119,28 +126,29 @@ def main(fabric: L.Fabric, data_dir: Path, checkpoint_dir: Path, out_dir: Path,
     train_data = torch.load(data_dir / "train.pt")
     val_data = torch.load(data_dir / "test.pt")
 
-    if not any((lora_query, lora_key, lora_value, lora_projection, lora_mlp, lora_head)):
+    if not any((cfg['lora_query'], cfg['lora_key'], cfg['lora_value'], 
+                cfg['lora_projection'], cfg['lora_mlp'], cfg['lora_head'])):
         fabric.print("Warning: all LoRA layers are disabled!")
     config = Config.from_name(
```

Comment:
I suggest renaming this to something like `model_config`, as the name shadowing with `cfg` will inevitably become confusing.

It would also be nice if you could just do something like `Config.from_dict(cfg)` to generate the model config. Might be worth moving the extraction logic over there.

Reply:
makes sense! Changing name also addresses the problem below, I'll do that!